content/doc/book/installing/linux.adoc

```diff
@@ -147,7 +139,7 @@ sudo wget -O /etc/yum.repos.d/jenkins.repo \
 sudo rpm --import https://pkg.jenkins.io/redhat-stable/jenkins.io.key
 sudo dnf upgrade
 # Add required dependencies for the jenkins package
-sudo dnf install chkconfig java-devel
+sudo dnf install fontconfig java-11-openjdk
```

Comment:
I wonder if it can use the headless JRE package or if that fails.

Reply:
Not sure, but switching our tests and docs to the headless package seems better left to a separate PR, starting with `jenkinsci/packaging`. This PR isn't trying to use a perfect set of package name. It's just trying to be consistent with the other docs and tests.

Reply:
What's the right way to see if that's possible? JIRA issue? GH issue?

Reply:
Why don't you start by opening a PR in `jenkinsci/packaging` to switch the tests to the headless package in `molecule/default/install-deb.yml`/`templates/header.debian.html`, `molecule/default/install-rpm.yml`/`templates/header.redhat.html`, and `molecule/default/install-suse.yml`/`templates/header.opensuse.html`. If and when that PR is merged, the documentation in this repository can be updated to match.

Reply:
That does ring some bell